src/node.js

```diff
@@ -388,6 +396,53 @@
       nextTickQueue.push(obj);
       tickInfo[kLength]++;
     }
+
+    // Internal callbacks that must be run when there is no more micro-tasks
+    // scheduled for this turn.
+    function lateTick(callback) {
+      lateTickQueue.push(callback);
+      scheduleMicrotasks();
```

Comment:
@vkurchatkin to make sure that the microtask queue would be scheduled for consumption in case it wasn't when this function is called.